aiotfm/client.py

```diff
@@ -56,7 +56,8 @@ def __init__(self, community=0, loop=None):
 
 		self.room = None
 		self.trade = None
-		self.trades = []
+		self.trades = {}
+		self.pending_trades = [] # Trades without pid

```

Comment:
This list has no .append call.

Reply:
I removed `Client.pending_trades` and it throws an error if the player cannot be resolved into a pid.